impeller/compiler/impellerc_main.cc

```diff
@@ -114,7 +114,10 @@ bool Main(const fml::CommandLine& command_line) {
         std::cerr << "Runtime stage information was nil." << std::endl;
         return false;
       }
-      auto stage_data_mapping = stage_data->CreateMapping();
+
+      auto stage_data_mapping = switches.use_raw_output
```

Comment:
This is a testing hack. Pending the decision for what format to use, we will either leave this as is or add JSON support.